03-components/20-UiClock/UiClock.js

```diff
@@ -1,9 +1,31 @@
-import { defineComponent } from 'vue'
+import { defineComponent, ref, onMounted, onUnmounted } from 'vue'
 
 export default defineComponent({
   name: 'UiClock',
 
-  setup() {},
+  setup() {
+    const getTime = () => {
+      return new Date().toLocaleTimeString(navigator.language, { timeStyle: 'medium' })
+    }
 
-  template: `<div class="clock">10:12:02</div>`,
+    const id = ref(0)
```

Comment:
`id` can be a regular variable, not a `ref`. It doesn't require reactivity - it is not used in rendering (template), there is no reaction on its change.

Reply:
Remembered. Fixed.